system/Commands/Generators/Views/model.tpl.php

```diff
@@ -17,6 +17,8 @@ class {class} extends Model
     protected $protectFields    = true;
     protected $allowedFields    = [];
 
+    protected bool $updateOnlyChanged = true;
```

Comment:
Maybe for better usability, we should default this to false?

Reply:
I think the default to `false` is better.
So if we agree, I will change it.

It changes the update behavior, but existing apps do not seem to break.
So it is not a breaking change?

Reply:
I don't believe it would have side effects on existing apps. We should mention this change but not under "breaking changes".

+1 for `false`

Reply:
Since it's only for new models generated I wouldn't consider it a breaking change of any sort. Agree with @michalsn  that maybe mention it, but not under breaking changes.

Reply:
Oh, you mean only change the value in the `make:model` template.
I thought you meant both BaseModel and the template.
Isn't it confusing to change only the template?